u7-components/src/router/index.ts

```diff
@@ -0,0 +1,41 @@
+import Vue from 'vue'
+import VueRouter, { RouteConfig } from 'vue-router'
+import Home from '../views/Home.vue'
+import VideoPlayer from '../views/VideoPlayer.vue'
+import BookSearch from '../views/BookSearch.vue'
+import BookDetails from '../views/BookDetails.vue'
```

Comment:
คุณสามารถใช้ `@` เพื่อข้ามไปยังไดเร็กทอรี `src` ฉันชอบใช้พาธ "ฐานญาติ" เช่น
You can use `@` to jump to the `src` directory. I prefer to use "base-relative" paths, for example:
```ts
import Home from '@/views/Home.vue'
```